### ISSUE TITLE
ValueError at make_beadplots stage

### ISSUE THREAD

**Issue by ArtPoon** [PoonLab/covizu#206]
```console
🏄 [3:36:19.747932] start B.1.1.70, 679 entries
🏄 [3:36:20.957782][0/36] loading JSON
[ ... ]
🏄 [3:36:25.788906][2/36] Reduced to 331 variants; generating feature set union
🏄 [3:36:25.789371][4/36] Writing distance matrix to temporary file
[ ... ]
🏄 [3:36:28.897609][8/36]   row 0 of 331
Traceback (most recent call last):
  File "batch.py", line 249, in <module>
    result = make_beadplots(by_lineage, args, cb.callback, t0=cb.t0.timestamp())
  File "batch.py", line 206, in make_beadplots
    label_dict = import_labels(handle)
  File "batch.py", line 161, in import_labels
    qname, idx = line.strip('\n').split(',')
ValueError: too many values to unpack (expected 2)
```

**Comment by ArtPoon** [comment]
Smells like a sequence label contains a `,` character.

Yup.  This one is the culprit:
```python
>>> handle = open('data/B.1.1.70.labels.csv')
>>> for line in handle:
...   items = line.strip().split(',')
...   if len(items) != 2:
...     print(line)
... 
hCoV-19/Hungary/US-32533w,/2020|EPI_ISL_677751|2020-09-02,297

>>> 
```

**Comment by ArtPoon** [comment]
Hot fix to master branch, commit `06e685e0e756b9caf11a0f2af6cef718970a48ee`
